build/dependencies.props

```diff
@@ -4,6 +4,7 @@
     <CoreFxVersion>4.3.0</CoreFxVersion>
     <InternalAspNetCoreSdkVersion>2.1.0-*</InternalAspNetCoreSdkVersion>
     <NETStandardImplicitPackageVersion>$(BundledNETStandardPackageVersion)</NETStandardImplicitPackageVersion>
+    <NETStandardLibraryNETFrameworkVersion>2.0.0-*</NETStandardLibraryNETFrameworkVersion>
```

Comment:
I think instead of pinning to a specific version, I could use the `BundledNETStandardPackageBVersion` here instead?
https://github.com/aspnet/CoreCLR/blob/dev/tools/MirrorLatestCoreCLR/restore-projects/NetStandard2.proj#L6